### PR TITLE
Feature/polish

### PR DESCRIPTION
**Pull request by TylerRockwell**
### Why?
https://github.com/smashingboxes/apprenticeship/tree/master/Curriculum/Backend/bookstore
Because I may not be an artist, but I can at least get nicely formatted data to the screen

### What Changed?
Lots of small style tweaks, like font weight, money being shown as currency, etc...
Most popular sort and the search feature work together now

### CODE REVIEW

File: app/controllers/books_controller.rb [hunk 1]
```diff
@@ -4,11 +4,14 @@ class BooksController < ApplicationController
...
+                       .page(params[:page])
+
     else
       @books = Book.search(params[:search])
                    .order_by(params[:sort_field], params[:sort_order])
                    .page(params[:page])
+                   .decorate
     end
     @sortable_fields = Book.sort_options
   end
```
Comment (jahammo2) [inline app/controllers/books_controller.rb]: You don't need to change anything but as you start using services, you'll want to do all your sorting/decorating/etc logic within the service. Just an FYI.

Comment (TylerRockwell) [inline app/controllers/books_controller.rb]: Interesting. I'm assuming I'll have more services in the future.

**Comment by jahammo2** [comment]
2 things:
1. You probably won't be doing much styling as a back end dev here so I don't want to harp on your styling/class naming. But if you do find yourself doing more and more styling, you should read and follow the [SB Guide](http://wiki.smashingboxes.com/articles/sb-css-scss-style-guide) and learn about [BEM](https://css-tricks.com/bem-101/). Don't change anything now because it's probably not worth your time. Furthermore, what you have is definitely not bad.
2. I've never used/made decorators nor have I used that Draper gem so you'll probably want another set of :eyes: on this PR.

**Comment by jahammo2** [comment]
Oh and it looks good. If you can't get more :eyes:, it should be fine to :shipit:

**Comment by TylerRockwell** [comment]
Thanks for looking things over. I'll hopefully be avoiding more css in the future, but I'll read over the links you posted anyway because I'm sure it's good knowledge to have regardless. I'll see if I can find someone to check out the decorators. Thanks, @jahammo2 !

**Comment by albaer** [comment]
This looks good to me, but I also have not used the draper gem so if you're looking for feedback on that you should hit up someone else (I believe Josh has used it a fair amount, but I'm not sure where I'm getting that from). Considering where you are in the project though, I'm good with you :ship: ing from here and focusing on Taperole for now.

**Comment by TylerRockwell** [comment]
Thanks, @albaer !

**Pull request merged**